symbolic-symcache/src/lib.rs

```diff
@@ -1,4 +1,101 @@
-//! Provides SymCache support.
+//! Provides SymCache support. This includes a reader and writer for the binary format, as well as
+//! helper traits and functions to apply transformations to debugging symbols before they are
+//! written to the SymCache.
```

Comment:
nit: it is preferable to have a single-line summary first just like for git commits.  The generated docs will use the first line in the module overview parts and this is much nicer to read if its not the start of a paragraph.